packages/ui/src/components/shared/export/Json.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { AccountBackup } from '@coong/keyring/types';
+import { Download } from '@mui/icons-material';
+import { Button } from '@mui/material';
+import FileSaver from 'file-saver';
+import { ExportObject, Props } from 'types';
+
+interface JsonProps extends Props {
+  value: AccountBackup;
```

Comment:
Similar to the `QrBackup` type, we should create a `JsonBackup` for this case as well, instead of passing the raw` AccountBackup`, otherwise there's no need to have the `object` prop since we meant an account backup by using the `AccountBackup` type for the value prop above.